migrations/V80__Add_Get_Needed_Function.SQL

```diff
@@ -0,0 +1,98 @@
+CREATE OR REPLACE FUNCTION GET_NEED_NEEDED( table_name TEXT, model_plan_idParam UUID, h_changed HSTORE ) --TODO: we could make this take an array if want
```

Comment:
The name is a bit confusing. Some alternatives might be DETERMINE_REQUIRED_NEED or GET_REQUIRED_NEED. What do you think? Do you have any other ideas?

Reply:
That makes sense to me! Thanks for the suggestion @OddTomBrooks , I'll update it

Reply:
I settled on `DETERMINE_SECTION_NEEDS`